reflex/components/plotly/plotly.py

```diff
@@ -43,3 +43,12 @@ class Plotly(PlotlyLib):
 
     # If true, the graph will resize when the window is resized.
     use_resize_handler: Var[bool]
+
+    def _render(self):
+        tag = super()._render()
+        if self.layout is None:
+            tag.remove_props("data", "layout")
+            tag.special_props.add(Var.create_safe(f"{{...{self.data}}}"))
```

Comment:
```suggestion
            tag.special_props.add(Var.create_safe(f"{{...{self.data._var_name_unwrapped}}}"))
```

this might fix tom's issue